core/server/controllers/frontend.js

```diff
@@ -564,8 +565,9 @@ frontendControllers = {
                                 // make sure URL has a trailing slash
                                 try {
                                     parsed = url.parse(attributeValue);
+                                    path = (parsed.pathname) ? parsed.pathname : null;
 
-                                    if (parsed.pathname && parsed.pathname.slice(-1) !== '/') {
+                                    if (path && path.slice(-1) !== '/' && path.indexOf('content/images/') === -1) {
```

Comment:
I see a problem with this solution, in that it will only work for locally hosted images. If I add a URL linking to an image elsewhere, the slash gets added. I am not sure that we need to add slashes to these urls at all cc @jaswilli ? If it's a Ghost URL, it'll redirect to the slashed version, and if it's an external URL then we also shouldn't be adding a slash?